db/migrate/20160321161136_add_devise_to_users.rb

```diff
@@ -0,0 +1,49 @@
+class AddDeviseToUsers < ActiveRecord::Migration
+  def self.up
+    change_table(:users) do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ""
+      t.string :encrypted_password, null: false, default: ""
```

Comment:
Does Devise / ActiveAdmin handle user authentication, then? Had been planning on setting up the User model with ActiveModel's `has_secure_password` attribute using a `password_digest` field, but it seems like this might cover that. I'm not familiar with Devise/ActiveAdmin, though, so I'm not sure.